aio/content/examples/upgrade-phonecat-2-hybrid/app/phone-detail/phone-detail.template.html

```diff
@@ -1,7 +1,7 @@
 <!-- #docregion -->
 <div *ngIf="phone">
   <div class="phone-images">
-    <img [src]="img" class="phone" [alt]="'phone ' + index"
+    <img [src]="img" class="phone" [alt]="'Phone ' + phone.name + ' - thumbnail ' + index"
```

Comment:
Nit: I find that the attribute binding would be a little more readable here (and below):

```suggestion
    <img [src]="img" class="phone" alt="Phone {{ phone.name }} - thumbnail {{ index }}"
```